src/mono/wasm/Wasm.Build.Tests/TestAppScenarios/AppTestBase.cs

```diff
@@ -104,13 +83,13 @@ void OnConsoleMessage(IConsoleMessage msg)
                 throw new Exception(msg.Text);
 
             if (options.OnConsoleMessage != null)
-                options.OnConsoleMessage(msg, page);
+                options.OnConsoleMessage(msg);
         }
 
-        TimeSpan timeout = TimeSpan.FromMinutes(2);
-        await Task.WhenAny(tcs.Task, Task.Delay(timeout));
-        if (!tcs.Task.IsCompleted)
-            throw new Exception($"Timed out after {timeout.TotalSeconds}s waiting for process to exit");
+        //TimeSpan timeout = TimeSpan.FromMinutes(2);
```

Comment:
remove dead code ?

Reply:
The timeout was there only to terminate the test when the browser doesn't finish with exit code